docs/faq.rst

```diff
@@ -23,8 +23,15 @@ and then add ``permalink: /404.html`` in the `YAML front matter`_.
 
 If you are using the github provided domain, make sure to set the :confval:`notfound_urls_prefix` to your repository's name in between two forward slashes. For example if your repository is named ``MyRepo``, then ``notfound_urls_prefix = "/MyRepo/"``.
 
+Does this extension work with `Jupyter Book <https://jupyterbook.org/intro.html>`_?
+-------------------------------------------
+
+Yes.
+
+You will need to install the extension locally and add it to your Jupyter Book ``_config.yml`` as an extra extension, and you will likely need to set the :confval:`notfound_urls_prefix` to your repository's name in between two forward slashes. For example if your repository is named ``MyRepo``, then ``notfound_urls_prefix = "/MyRepo/"``.
```

Comment:
> and you will likely need to set the :confval:`notfound_urls_prefix` to your repository's name in between two forward slashes. For example if your repository is named ``MyRepo``, then ``notfound_urls_prefix = "/MyRepo/"``.

If I understand correctly, this is required to be like you described because your repository is on GitHub pages, not because of Jupyter Book. See https://sphinx-notfound-page.readthedocs.io/en/latest/faq.html#does-this-extension-work-with-github-pages

So, I think it may be worth clarifying this by mentioning it and linking to the other FAQ. What do you think?

BTW, your docs at https://melaniewalsh.github.io/Intro-Cultural-Analytics/ look really nice 😃

Reply:
Hmmm this is a good point. I wonder how the extension works for a Jupyter Book that doesn't use GitHub Pages? I think the main Jupyter Book docs use Netlify, so maybe that could be a test case (@choldgraf?).

Maybe I will wait to clarify until I know more about how Jupyter Books works without GitHub Pages.

Also thanks for the kind words, @humitos! 😊